### PR TITLE
[IMP] account_edi_facturx: import shipping partner

### PR DESCRIPTION
**Pull request by bfr-o**
When importing a facturx file, the shipping_partner_id on account.move is now processed if it's present in the xml.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by smetl** [comment]
@robodoo r+

File: addons/account_edi_facturx/models/account_edi_format.py [hunk 1]
```diff
@@ -192,17 +192,19 @@ def _import_facturx(self, tree, invoice):
...
-                partner_name = elements and elements[0].text
-                partner = elements and self.env['res.partner'].search([('name', 'ilike', partner_name)], limit=1)
-            if not partner:
-                elements = tree.xpath('//ram:' + partner_type + '//ram:URIID[@schemeID=\'SMTP\']', namespaces=tree.nsmap)
-                partner = elements and self.env['res.partner'].search([('email', '=', elements[0].text)], limit=1)
-            if partner:
-                invoice_form.partner_id = partner
+            invoice_form.partner_id = self._retrieve_partner(
+                name=self._find_value('//ram:' + partner_type + '/ram:Name', tree, namespaces=tree.nsmap),
+                mail=self._find_value('//ram:' + partner_type + '//ram:URIID[@schemeID=\'SMTP\']', tree, namespaces=tree.nsmap),
```
Comment (william-andre) [inline addons/account_edi_facturx/models/account_edi_format.py:197]:
```suggestion
                mail=self._find_value(f"//ram:{partner_type}//ram:URIID[@schemeID='SMTP']", tree, namespaces=tree.nsmap),
```

Didn't review functionally as @smetl is more aware than me, but just this suggestion :)

Comment (smetl) [reply addons/account_edi_facturx/models/account_edi_format.py:197]: Arf too late... I like this suggestion... :/

Comment (bfr-o) [reply addons/account_edi_facturx/models/account_edi_format.py:197]: Done in the fw-port since there was a conflict anyway.

**Pull request closed without merging**